### PR TITLE
Update library dependencies

### PR DESCRIPTION
**Pull request by mikucionisaau**

### CODE REVIEW

File: cmake/UUtils.cmake [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
     set(UUtils_WITH_TESTS OFF CACHE BOOL "UUtils tests")
     set(UUtils_WITH_BENCHMARKS OFF CACHE BOOL "UUtils benchmarks")
     FetchContent_Declare(
             UUtils
-            GIT_REPOSITORY https://github.com/mikucionisaau/UUtils.git
-            GIT_TAG cmake-alias
-            #GIT_REPOSITORY https://github.com/UPPAALModelChecker/UUtils.git
-            #GIT_TAG v2.0.2
+            #GIT_REPOSITORY https://github.com/mikucionisaau/UUtils.git
+            #GIT_TAG cmake-alias
```
Comment (thorulf4) [inline cmake/UUtils.cmake:16]: Please remove these

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -3,19 +3,20 @@ add_library(UDBM STATIC DBMAllocator.cpp dbm.c fed_dbm.cpp mingraph.c mingraph_r
...
         # where the library itself will look for its internal headers
-        ${CMAKE_CURRENT_BINARY_DIR}/../include
-        ${CMAKE_CURRENT_SOURCE_DIR}/../include
+        ${PROJECT_BINARY_DIR}/include
+        ${PROJECT_SOURCE_DIR}/include
     PUBLIC
         # where top-level project will look for the library's public headers
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/../include>
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
+        $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
```
Comment (thorulf4) [inline src/CMakeLists.txt:18]: It doesn't really make sense to add the `.._BINARY_DIR` to include directories. Try removing 
`${PROJECT_BINARY_DIR}/include`
and
`$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>`

**Review by thorulf4 — CHANGES_REQUESTED** [review]
Minor cmake changes

File: cmake/UUtils.cmake (hunk 1 above)
Comment (mikucionisaau) [reply cmake/UUtils.cmake:16]: I can surely remove, but it is very handy to have for testing ;-)

File: src/CMakeLists.txt (hunk 2 above)
Comment (mikucionisaau) [reply src/CMakeLists.txt:18]: Nice catch!

**Review by thorulf4 — APPROVED** [review]

**Pull request merged**